expo/src/api/GnoNativeApi.ts

```diff
@@ -116,25 +119,36 @@ export class GnoNativeApi implements GnoKeyApi, GoBridgeInterface {
   }
 
   async setRemote(remote: string): Promise<SetRemoteResponse> {
-    const client = await this.#getClient();
-    const response = await client.setRemote(new SetRemoteRequest({ remote }));
+    const client = this.#getClient();
+    const response = client.setRemote(new SetRemoteRequest({ remote }));
     return response;
   }
 
   async getRemote(): Promise<string> {
-    const client = await this.#getClient();
+    const client = this.#getClient();
     const response = await client.getRemote(new GetRemoteRequest());
     return response.remote;
   }
 
+  async signTx(
+    txJson: string,
+    address: Uint8Array,
+    accountNumber: bigint,
```

Comment:
accountNumber and sequenceNumber should default to 0.  This is the normal case.
https://github.com/gnolang/gnonative/blob/ecdd187838f1e60acd036daadd07c4414ba6ffeb/api/gnonativetypes/gnonativetypes.go#L319-L322

Reply:
Ok, so you want that I remove the two parameters so the front-end cannot set them and I hard code to 0?

Reply:
No, don't remove them. Use parameter default values `accountNumber: bigint = 0, sequenceNumber: bigint = 0` . (Maybe you have to make 0 into a bigint constant. I don't know how to do that.)

Reply:
oh you want to make them optional! Understand, I'll do that :)

Reply:
done